webhook_server/libs/handlers/runner_handler.py

```
@@ -742,3 +777,89 @@ async def cherry_pick(self, pull_request: PullRequest, target_branch: str, revie
+
+        retrigger_config = self.github_webhook.retrigger_checks_on_base_push
+
+        if retrigger_config == "all":
+            checks_to_run = available_checks
+        elif isinstance(retrigger_config, list):
+            # Filter to only configured checks that are available
+            checks_to_run = [check for check in retrigger_config if check in available_checks]
+            if not checks_to_run:
+                self.logger.warning(
+                    f"{self.log_prefix} None of the configured retrigger checks {retrigger_config} "
+                    f"are available. Available: {available_checks}"
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)